creusot-contracts/src/logic/int.rs

```diff
@@ -164,68 +165,149 @@ impl Int {
     }
 }
 
-#[cfg(creusot)]
-impl Add<Int> for Int {
-    type Output = Int;
+impl AddLogic for Int {
+    type Output = Self;
+    #[logic]
     #[trusted]
     #[creusot::no_translate]
     #[creusot::builtins = "add_int"]
-    fn add(self, _: Int) -> Self {
-        panic!()
+    fn add(self, other: Self) -> Self {
+        let _ = other;
```

Comment:
```suggestion
    fn add(self, _: Self) -> Self {
```

Reply:
ahhh, this is so it renders better in doc :)

Reply:
Well, then, you can use `#[allow(unused)]`.